### PR TITLE
fix: add test for null personal identity number

### PR DESCRIPTION
**Pull request by mida-digg**
# Pull Request Description

When creating a new account, email and personal identity number are now optional.

Fixes #(issue)

## Checklist

- [x] Changes are limited to a single goal (avoid scope creep)
- [x] I confirm that I have read any Contribution and Development guidelines (CONTRIBUTING and DEVELOPMENT) and are following their suggestions.
- [x] I confirm that I wrote and/or have the right to submit the contents of my Pull Request, by agreeing to the _Developer Certificate of Origin_, (adding a 'sign-off' to my commits).

### CODE REVIEW

File: src/test/java/se/digg/wallet/ecosystem/WalletClientGatewayTest.java [hunk 1]
```diff
@@ -133,6 +142,16 @@ private static String stubAccountV0Request(ECKey ecKey) {
         }""".formatted(id, ecKey.toPublicJWK().toJSONString());
   }
 
+  private static String stubAccountV0RequestWithNullPersonalIdentityNumber(ECKey ecKey) {
+    return """
+        {
+          "personalIdentityNumber": "%s",
```
Comment (nickjann-04) [inline src/test/java/se/digg/wallet/ecosystem/WalletClientGatewayTest.java]: Här blir "null" en sträng. Borde bara vara null.

Comment (mida-digg) [reply src/test/java/se/digg/wallet/ecosystem/WalletClientGatewayTest.java]: Fort men slarvigt! :-)

**Review by erik-eriksson34 — DISMISSED** [review]

**Review by nickjann-04 — DISMISSED** [review]

**Review by nickjann-04 — APPROVED** [review]
